### DISCUSSION TITLE
Multiline textbox?

### DISCUSSION THREAD

**Discussion by how-to-do-frontend** [ppy/osu-framework#6333] [Ideas]
I know this has been suggested before, but nothing's been said in 4 years
Just wondering if this is planned soon

**Comment by Joehuu** [comment]
As mentioned in https://github.com/ppy/osu-framework/issues/1618#issuecomment-682527119 four years ago, it was a low priority.

I believe it is still a low priority unless there is a use case for the osu!(lazer) client exclusively (i.e. something not on the website). For example, you can't write paragraphs in comments, but you can do so in the website.
